automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -471,6 +482,176 @@ class RawlsApiSpec extends TestKit(ActorSystem("MySpec")) with FreeSpecLike with
+            One example is the Broad's copy: gs://gcp-public-data--broad-references/hg38/v0/hapmap_3.3.hg38.vcf.gz
+
+            Also if/when we switch to multi-cloud, this file should be publicly available on other clouds, such as
+            https://registry.opendata.aws/broad-references/
+             */
+            val entityType = "participant"
+            val entityId = "hapmap_vcf"
+            val vcfPath =  "gs://genomics-public-data/resources/broad/hg38/v0/hapmap_3.3.hg38.vcf.gz"
+            val vcfIndexPath =  "gs://genomics-public-data/resources/broad/hg38/v0/hapmap_3.3.hg38.vcf.gz.tbi"
+
+            val entityTsv =
+              s"""|entity:${entityType}_id\tvcf_path\tvcf_index_path
+                  |$entityId\t$vcfPath\t$vcfIndexPath
+                  |""".stripMargin
+
+            val inputs = Map(
+              "test_count_variants.indexedVcf" ->
+                """{
+                  |  "vcf": this.vcf_path,
+                  |  "vcf_idx": this.vcf_index_path
+                  |}
+                  |""".stripMargin,
+              "test_count_variants.interval" -> """"chr20"""",
+            )
+
+            val outputs = Map(
+              "test_count_variants.count" -> "this.chr20_vcf_count",
+            )
+
+            Orchestration.methodConfigurations.createMethodConfigInWorkspace(
+              wsNs = projectName,
+              wsName = workspaceName,
+              method = privateMethod,
+              configNamespace = SimpleMethodConfig.configNamespace,
+              configName = SimpleMethodConfig.configName,
+              methodConfigVersion = SimpleMethodConfig.snapshotId,
+              inputs = inputs,
+              outputs = outputs,
+              rootEntityType = entityType,
+            )
+
+            Orchestration.importMetaData(
+              ns = projectName,
+              wsName = workspaceName,
+              fileName = "entities",
+              fileContent = entityTsv,
+            )
+
+            // it currently takes ~ 5 min for google bucket read permissions to propagate.
+            // We can't launch a workflow until this happens.
+            // See https://github.com/broadinstitute/workbench-libs/pull/61
+            // and https://broadinstitute.atlassian.net/browse/GAWB-3327
+
+            Orchestration.workspaces.waitForBucketReadAccess(projectName, workspaceName)
+
+            val submissionId = Rawls.submissions.launchWorkflow(
+              billingProject = projectName,
+              workspaceName = workspaceName,
+              methodConfigurationNamespace = SimpleMethodConfig.configNamespace,
+              methodConfigurationName = SimpleMethodConfig.configName,
+              entityType = entityType,
+              entityName = entityId,
+              expression = "this",
+              useCallCache = false,
+              deleteIntermediateOutputFiles = false,
+            )
+            // clean up: Abort submission
```

Comment:
nit: does this comment really add anything?

Reply:
This was copypasta from the original test. I'm fine if this is removed @salonishah11

Reply:
I have removed the comment